.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Vue Test Utils Contributing Guide
+
+Hi! I’m really excited that you are interested in contributing to Vue Test Utils. Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
+
+- [Code of Conduct](https://github.com/vuejs/vue/blob/dev/.github/CODE_OF_CONDUCT.md)
+- [Issue Reporting Guidelines](#issue-reporting-guidelines)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [Development Setup](#development-setup)
+- [Project Structure](#project-structure)
+
+## Issue Reporting Guidelines
+
+- Always use [https://new-issue.vuejs.org/](https://new-issue.vuejs.org/) to create new issues.
+
+## Pull Request Guidelines
+
+- The `master` branch is basically just a snapshot of the latest stable release. All development should be done in dedicated branches. **Do not submit PRs against the `master` branch.**
+
+- Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
+
+- Work in the `src` folder and **DO NOT** checkin `dist` in the commits.
+
+- It's OK to have multiple small commits as you work on the PR - we will let GitHub automatically squash it before merging.
+
+- Make sure `npm test` passes. (see [development setup](#development-setup))
+
```

Comment:
I think it is better to add below.

As general rule, [cosmetic change](http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code) is not accepted.
Please do not send a pull request which is cosmetic change if you do not understand specifications and codes.